src/SLCore/Common/Helpers/ConnectionIdHelper.cs

```diff
@@ -33,6 +35,22 @@ internal class ConnectionIdHelper : IConnectionIdHelper
         private const string SonarQubePrefix = "sq|";
         private static readonly Uri SonarCloudUri = new Uri("https://sonarcloud.io");
 
+        public string GetConnectionIdFromUri(string uriString, string organisation)
```

Comment:
BoundSonarQubeProject has `Uri` property, why do you use `string` here? The other method of this class also returns Uri and not string